### PR TITLE
release: Add notes for v1.0.0-beta2

### PR DESCRIPTION
**Pull request by marclop**
## Description

Generates the release notes for v1.0.0-beta2

## Types of Changes

- [x] Documentation

### CODE REVIEW

**Comment by marclop** [comment]
Yeah, I'll add it soon

File: notes/v1.0.0-beta2.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Changelog
+
+## Notable changes
+<!-- Add the list of notable changes, remove otherwise.-->
+This is a slim release on the SDK side, mainly contains bug fixes, version updates development improvements.
```
Comment (nrichers) [inline notes/v1.0.0-beta2.md]: I would perhaps say `This release contains bug fixes, version updates, and development improvements.` (Keep it positive--saying it's slim might be seen to be a negative thing. If you don't have to say it, I'd just omit.)

File: notes/v1.0.0-beta2.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Changelog
```
Comment (nrichers) [inline notes/v1.0.0-beta2.md]: Note this file name still has a `v` in it. ^^

File: notes/v1.0.0-beta2.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+<!-- Add the list of notable changes, remove otherwise.-->
+This is a slim release on the SDK side, mainly contains bug fixes, version updates development improvements.
+As usual, the autogenerated Go code has been updated to match the 2.5.0 ECE API swagger specification.
+
+### Features
+
+#### Customizable User-Agent
+
+Allows the user to configure a user agent via `api.Config.UserAgent`. If the field is not set, it defaults
+to `cloud-sdk-go/<version>`, which after this release it will be `cloud-sdk-go/2.5.0`.
```
Comment (karencfv) [inline notes/v1.0.0-beta2.md]:
```suggestion
to `cloud-sdk-go/<version>`, which after this release will be `cloud-sdk-go/2.5.0`.
```

File: notes/v1.0.0-beta2.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+<!-- Add the list of notable changes, remove otherwise.-->
+This is a slim release on the SDK side, mainly contains bug fixes, version updates development improvements.
+As usual, the autogenerated Go code has been updated to match the 2.5.0 ECE API swagger specification.
+
+### Bug fixes
+<!-- Add the list of bug fixes, remove otherwise.-->
+
+#### Plan tracker reported some plans as successfully finished when they actually failed
+
+Fixes a bug in the plan tracker logic which made it return the first
```
Comment (nrichers) [inline notes/v1.0.0-beta2.md]: `more notably` --> `most notably`

Maybe change `set a cluster vacate as "Sucessful"` --> `flag a cluster vacate as sucessful` (if that doesn't change the meaning of the sentence)

File: notes/v1.0.0-beta2.md (hunk 3 above)
Comment (karencfv) [inline notes/v1.0.0-beta2.md]: 😄 
```suggestion
to `cloud-sdk-go/<version>`, which after this release will be `cloud-sdk-go/2.5.0`.
```

**Pull request merged**